iml-gui/crate/src/breakpoints.rs

```diff
@@ -0,0 +1,47 @@
+//! # Breakpoints
+//!
+//! This module lists the existing media breakpoints we use and contains some helper functions for working with them.
+//!
+//! It should be kept in sync with any breakpoint changes to `tailwind.config.js`.
+
+use seed::window;
+
+pub(crate) const SM: f64 = 569.;
+pub(crate) const MD: f64 = 769.;
+pub(crate) const LG: f64 = 1025.;
+pub(crate) const XL: f64 = 1701.;
+
+#[derive(Clone, Copy, Debug, Eq, PartialEq, Ord, PartialOrd)]
+pub(crate) enum Size {
+    XS,
```

Comment:
I wonder if this could be `enum Size { XS = 0, SM = 569,...}` and so on.

Reply:
Hrrm, didn't consider that, looking.

Reply:
So, this *does* work:

```rust
#[derive(Clone, Copy, Debug, Eq, PartialEq, Ord, PartialOrd)]
pub(crate) enum Size {
    XS = 0,
    SM = 569,
    MD = 769,
    LG = 1025,
    XL = 1701,
}
```

However any usage of the enum then needs to be double casted to compare with the f64 of `inner_width`:

`w >= Size::XL as u64 as f64`

Not sure it's worth the cost of callers having to do that each time over the current `const` bindings.

Reply:
Well, then update `inner_width` to return an int. Width is integer after all.

Reply:
yeah, ok.